### ISSUE TITLE
extract the trained classification head layer (with parameters)

### ISSUE THREAD

**Issue by batselem** [pytorch/fairseq#2943]
How can I use last classifier layer on a different model? 
I am going to replicate the classifier layer trained on top of roberta. I extract features from [CLS] token's output and trained a linear layer classification based on these features. But I couldn't get as high accuracy as the roberta does. For example, roberta trained on commonsenseqa obtained 76% accuracy on valid set (reported accuracy is 78%) but when I extract the features from CLS last layer and feed it to a neural network model. After training it, I got 71% accuracy. Thank you.

**Comment by batselem** [comment]
Just solved it. 

logits = roberta.model.classification_heads['sentence_classification_head'](data)
